packages/addon/package.json

```diff
@@ -37,10 +38,7 @@
     "./preset.js": {
       "default": "./preset.js"
     },
-    "./register.js": {
```

Comment:
Why remove this?

Reply:
I'm removing the requirement for the register file, it will just check if it exists or not instead of always trying to load it

Reply:
I think since the file exists and it is published then it should be accessible, if it is removed here then the file should also be removed and not published

Reply:
yeah I removed the register file